packages/preview/sweet-graduate-resume/0.1.0/lib_fontawesome.typ

```diff
@@ -0,0 +1,26 @@
+//! typst-fontawesome
+//!
+//! https://github.com/duskmoon314/typst-fontawesome
```

Comment:
Is there a reason you copied the contents of this package instead of importing it?

Reply:
I wasn't aware of typst having local imports as well when I started making/using this and later on just went with it. I hope it is not a problem since the relevant people have been acknowledged in the source files.

Reply:
No it is not a real problem, but it would be cleaner and easier for you to maintain to just use the actual package. If you prefer to keep it as is, it is possible though.

Reply:
I'll probably get around to do this when I have a minor release bump. Thanks for the suggestion.